goalc/compiler/compilation/Atoms.cpp

```diff
@@ -26,8 +27,44 @@ static const std::unordered_map<
         {".lvf", &Compiler::compile_asm_lvf},
         {".svf", &Compiler::compile_asm_svf},
         {".xor.vf", &Compiler::compile_asm_xor_vf},
+        // TODO - Readability wise, it would probably be nice to have all the varients like
+        // '.sub.xyzw.vf' and so on, but it's not required at this point and needs to be done in a
+        // clever way to avoid lots of duplication
+        //
+        // TODO - difficult to cut down duplication as the map wants a function _reference_.
+        // ie. - I'd like to just call a function that returns a function, but the path forward
+        // isn't obvious to me yet.
```

Comment:
This is somewhat intentionally constraining to keep things simple, so it's always very clear what part of the compiler is compiling a form. A previous iteration of the compiler had a more complicated way to parse forms and it was kind of a nightmare.